shpkpr/bluegreen.py

```diff
@@ -1,275 +0,0 @@
-# stdlib imports
-import logging
-import time
-from datetime import datetime
-
-# third-party imports
-import click
-import requests
-
-# local imports
-from shpkpr import exceptions
-
-
-logger = logging.getLogger(__name__)
-
-
-# amount of time (in seconds) between subsequent polls of the load balancer
-# during a blue/green deploy.
-MARATHON_LB_POLL_INTERVAL = 5
-
-
-class SwapApplicationTimeout(exceptions.ShpkprException):
-    pass
-
-
-def get_deployment_label(app):
-    return get_deployment_group(app) + "_" + app['labels']['HAPROXY_0_PORT']
-
-
-def _is_app_listener(app, listener):
-    return (listener.pxname == get_deployment_label(app) and
-            listener.svname not in ['BACKEND', 'FRONTEND'])
-
-
-def fetch_app_listeners(app, marathon_lb_client):
-    haproxy_stats = marathon_lb_client.fetch_stats()
-    return [l for l in haproxy_stats if _is_app_listener(app, l)]
-
-
-def waiting_for_listeners(new_app, old_app, listeners, haproxy_count):
```

Comment:
This was a weird check that just duplicated the more-specific checks that followed (checking for UP/MAINT listeners for the new/old apps) so I dropped it in the refactor.